tier4/Pilot.Auto
# Fix simple planning simulator

- Fix launch
 - Fix timer callback
 - Add default param

How to test

Launch simulator, 

```
ros2 launch simple_planning_simulator simple_planning_simulator.launch.xml
```

then, send `/initialpose` from Rviz. Check if the `tf` is published correctly.
